README.md

```diff
@@ -10,8 +10,10 @@ The main project documentation is on the [page for this project at the AgileVent
 _This is a project of [AgileVentures](http://www.agileventures.org), a non-profit organization dedicated to crowdsourced learning and project development._  
 
 
-[![Build Status](https://semaphoreci.com/api/v1/projects/6c49e6ca-8220-41c6-b04f-67ed15252723/1030083/badge.svg)](https://semaphoreci.com/shf-project/shf-project)
+Sempahore status: [![Build Status](https://semaphoreci.com/api/v1/projects/6c49e6ca-8220-41c6-b04f-67ed15252723/1030083/shields_badge.svg)](https://semaphoreci.com/shf-project/shf-project) 
+
 Travis C. I. [![Build Status](https://travis-ci.org/AgileVentures/LocalSupport.png)](https://travis-ci.org/AgileVentures/LocalSupport)
```

Comment:
(Haven't set the link to this project because it hadn't been set up yet.  that's why it's pointing to the wrong project.  Would be clearer if I just removed the URL altogether, though.)